src/components/schemas/fields.yaml

```diff
@@ -236,6 +236,149 @@ contains_operator:
     Value signifying the operator for the field
   example: IN
 
+expense_policy_rule_minimal:
+  type: object
+  properties:
+    id:
+      type: string
+      example: tpr1234abcd
+      description: This represents the id of the expense policy rule that is violated for this expense.
+    description:
+      anyOf:
+        - $ref: '#/description'
+      description: |
+        This represents the description of the expense policy rule that is violated for this expense.
+      example: Receipt is mandatory for expense above $50
+    action_show_warning:
+      type: boolean
+      description: |
+        A true value here means that policy calculation suggests showing a warning when the user attempts to create an expense that violates this expense policy rule.
+      example: True
+      
+expense_policy_violating_expenses_query_params:
+  type: object
+  description: |
+    An expense may be violating an expense policy rule due to a group of other expenses. 
+    These query params can be used to fetch those "violating" expenses because of which this expense is causing a violation. <br>
```

Comment:
let's just drop the "violating"

too much violating word very nearby - can confuse the reader; let it be self explanatory